root/static/scripts/common/constants/locales.json

```diff
@@ -562,10 +590,10 @@
   "pa_Arab_PK":"Punjabi Pakistan Arabic",
   "pa_Guru":"Punjabi Gurmukhi",
   "pa_Guru_IN":"Punjabi India Gurmukhi",
+  "pcm":"Nigerian Pidgin",
+  "pcm_NG":"Nigerian Pidgin Nigeria",
   "pl":"Polish",
   "pl_PL":"Polish Poland",
-  "prg":"Prussian",
-  "prg_001":"Prussian World",
```

Comment:
Probably not in use right now - zero artist uses at least.

Reply:
It seems that you are doing twice the work that I already made in the linked ticket MBS-12527.

Reply:
I was, whoops :) Why wait to fix them though? It's not like they are working properly right now either :)

Reply:
Because it will be working properly once deployed in beta.